mmv1/products/spanner/BackupSchedule.yaml

```diff
@@ -0,0 +1,137 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BackupSchedule'
+base_url: projects/{{project}}/instances/{{instance}}/databases/{{database}}/backupSchedules
+self_link: 'projects/{{project}}/instances/{{instance}}/databases/{{database}}/backupSchedules/{{name}}'
+update_verb: :PATCH
+update_mask: true
+description: |
+  A backup schedule for a Cloud Spanner Database.
+  This resource is owned by the database it is backing up, and is deleted along with the database.
+  The actual backups are not though.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/spanner/docs/backup'
+  api: 'https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances.databases.backupSchedules'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  encoder: templates/terraform/encoders/spanner_schedule_backup.go.erb
+  decoder: templates/terraform/decoders/spanner_schedule_backup.go.erb
+  pre_create: templates/terraform/pre_create/spanner_schedule_backup.go.erb
+skip_sweeper: true
```

Comment:
Nit: could use a comment for why (presumably because they are swept when the database is swept)